app/errors.py

```diff
@@ -0,0 +1,20 @@
+class VaccineError(Exception):
+    """Parent class for vaccine errors"""
+
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
+
+
+class NotWearingMaskError(Exception):
+    """Don't have a mask"""
+
+    def __str__(self) -> str:
+        return "NotWearingMaskError"

```

Comment:
not descriptive

Reply:
write more informative string for this exception